publication-commons/src/main/java/no/unit/nva/publication/exception/GatewayTimeoutException.java

```diff
@@ -0,0 +1,16 @@
+package no.unit.nva.publication.exception;
+
+import nva.commons.apigateway.exceptions.ApiGatewayException;
+import org.apache.http.HttpStatus;
+
+public class GatewayTimeoutException extends ApiGatewayException {
```

Comment:
DbTimeoutException? ConsistencyTimeoutException?

Reply:
Hvis vi går for custom navn på exception istedet for å navngi exception til https status kodene, så tenker jeg at exceptionen bør ikke være en ApiGatewayException slik at vi uheldigvis lekker ut database internals.